vncviewer/org.tigervnc.vncviewer.metainfo.xml.in

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  (c) 2021-2022 Joachim Falk <joachim.falk@gmx.de>
+
+  Copying and distribution of this file, with or without modification, are
+  permitted in any medium without royalty provided the copyright notice and this
+  notice are preserved. This file is offered as-is, without any warranty.
+-->
+<component type="desktop-application">
+  <id>org.tigervnc.vncviewer</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>TigerVNC Viewer</name>
+  <summary>Remote desktop access using the VNC desktop sharing system</summary>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="1.12.0" date="2021-11-10">
+      <description>
+        <p>TigerVNC 1.12.0 is now available. Lots of changes have been made since the last release, but the highlights are:</p>
+        <ul>
+          <li>The native viewer now supports full screen over a subset of monitors (e.g. 2 out of 3), and reacts properly to monitors being added or removed</li>
+          <li>Recent server history in the native viewer</li>
+          <li>The native viewer now has an option to reconnect if the connection is dropped</li>
+          <li>Translations are now enabled on Windows and macOS for the native viewer</li>
+          <li>The native viewer now respects the system security policy (except when GnuTLS is statically linked)</li>
+          <li>Better handling of accented keys in the Java viewer</li>
+          <li>The Unix servers can now listen to both a Unix socket and a TCP port at the same time</li>
+          <li>The network code in both the servers and the native viewer has been restructured to give a more responsive experience</li>
+          <li>The vncserver service now correctly handles settings set to "0"</li>
+          <li>Fixed the clipboard Unicode handling in both the native viewer and the servers</li>
+          <li>Support for pointer "warping" in Xvnc and the native viewer, enabling e.g. FPS games</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+  <description>
+    <p>
+      The TigerVNC Viewer is a VNC client, which allows you to view a computing `desktop&apos; environment not only on the machine where it is running but from anywhere on the Internet and from a wide variety of machine architectures.
+      VNC stands for Virtual Network Computing, which is, in essence, a remote desktop sharing system.
+      This system is comprised of a VNC client (e.g., this software) and a VNC server, which shares the remote desktop.
+      These VNC servers are available for many operating systems, e.g., Linux, macOS, Windows, etc.
```

Comment:
Is this what we also have in the packages? It would probably be best to keep things in sync.

Reply:
I used my Debian package description for tigervnc-viewer. If you have a text you use to describe your vncviewer package, feel free to post and I will replace. And maybe also update the package description for the Debian tigervnc-viewer package.

Reply:
Under `contrib/` there are packages for both rpm and deb. Pick whichever you think is best, but it would be preferably that we end up with just one variant. :)

Reply:
Description has been updated with the one from contrib/packages/deb/ubuntu-focal/debian/control